timetracker/entries/views.py

```diff
@@ -106,3 +100,24 @@ def project_detail(request, pk):
         'project': project,
         'form': form,
     })
+
+class ProjectDetailCreateView(UpdateView):
+    """
+    View to update a Project instance and return
+    a response rendered by the template
+    """
+    model = Project
+    template_name = 'projects_detail.html'
+    form_class = ProjectForm
+    success_url = '/projects/'
+
+    def get_context_data(self, **kwargs):
+        context = super(ProjectCreateView, self).get_context_data(**kwargs)
+        context['projects'] = self.model.objects.all()
+        return context
+
+class RootRedirectView(RedirectView):
+    permanent = False
+
+    def get_redirect_url(self):
```

Comment:
You can use `redirect_url = reverse_lazy('client-list')` instead :wink: (`reverse_lazy` is at the same place as `reverse`)